packages/core/botpress/src/web/components/PluginInjectionSite/module.jsx

```diff
@@ -103,7 +103,8 @@ export default class InjectedModuleView extends React.Component {
     const bp = {
       events: EventBus.default,
       axios,
-      toast
+      toast,
+      router: this.context.router
```

Comment:
Sorry for my tenacity :) But I think it's not recommended to rely on this: https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/context.router.md
There's an alternative way to get access to the router, no?

Reply:
@emirotin , yes. ))
This means we don't even need to pass it as a props since `withRouter` can be used in the rendered modules. Removed that.